content/post/practical_ml.md

```diff
@@ -0,0 +1,77 @@
+---
+author: Ilan Man
+title: "Practical tips for real-world Data Science"
+publishDate: "2018-05-15"
```

Comment:
Publish date should be 5/13. I think I probably told you the wrong date originally. Sorry!

As a rule of thumb, we post on Sundays